src/ansys/dpf/core/operators/server/grpc_start_server.py

```diff
@@ -35,9 +35,18 @@ class grpc_start_server(Operator):
         client (able to send grpc calls) will
         be started
     data_sources : DataSources, optional
-        A data sources with result key 'grpc' and
-        file path port:ip can be used instead
-        of the input port and ip.
+        A data source with result key 'grpc' and file
+        path 'port:ip' can be used instead of
+        the input port and ip.
+    dpf_context : str or int, optional
+        This pin is associated with pin(2) = 2
+        (server started in a new process).
+        user can enter the integer associated
```

Comment:
```suggestion
        User can enter the integer associated
```

Reply:
@PProfizi This is autoformatted. The spec looks good on the server code